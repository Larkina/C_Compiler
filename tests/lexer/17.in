int main(){

	int a[100] = {0};
	int n = 0;
	scanf("%d", &n);
	for(int i = 0; i < n; ++i)	
		scanf("%d", &a[i]);
	for(int i = 0; i < n; ++i)
		for(int j = 0; j < n; ++j)
			swap(a[i], a[j]);
	for(int i = 0; i < n; ++i)
		printf("%d", a[i]);
}