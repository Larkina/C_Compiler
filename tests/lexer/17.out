value=int, text=int, type=KEY_WORD
value=main, text=main, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=), text=), type=R_PARENTHESIS
value={, text={, type=L_BRACE
value=int, text=int, type=KEY_WORD
value=a, text=a, type=VAR
value=[, text=[, type=L_BRAKET
value=100, text=100, type=INT
value=], text=], type=R_BRAKET
value==, text==, type=ASSIGN
value={, text={, type=L_BRACE
value=0, text=0, type=INT
value=}, text=}, type=R_BRACE
value=;, text=;, type=SEMICOLON
value=int, text=int, type=KEY_WORD
value=n, text=n, type=VAR
value==, text==, type=ASSIGN
value=0, text=0, type=INT
value=;, text=;, type=SEMICOLON
value=scanf, text=scanf, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=%d, text="%d", type=STRING
value=,, text=,, type=COMMA
value=&, text=&, type=AND_B
value=n, text=n, type=VAR
value=), text=), type=R_PARENTHESIS
value=;, text=;, type=SEMICOLON
value=for, text=for, type=KEY_WORD
value=(, text=(, type=L_PARENTHESIS
value=int, text=int, type=KEY_WORD
value=i, text=i, type=VAR
value==, text==, type=ASSIGN
value=0, text=0, type=INT
value=;, text=;, type=SEMICOLON
value=i, text=i, type=VAR
value=<, text=<, type=LT
value=n, text=n, type=VAR
value=;, text=;, type=SEMICOLON
value=++, text=++, type=INC
value=i, text=i, type=VAR
value=), text=), type=R_PARENTHESIS
value=scanf, text=scanf, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=%d, text="%d", type=STRING
value=,, text=,, type=COMMA
value=&, text=&, type=AND_B
value=a, text=a, type=VAR
value=[, text=[, type=L_BRAKET
value=i, text=i, type=VAR
value=], text=], type=R_BRAKET
value=), text=), type=R_PARENTHESIS
value=;, text=;, type=SEMICOLON
value=for, text=for, type=KEY_WORD
value=(, text=(, type=L_PARENTHESIS
value=int, text=int, type=KEY_WORD
value=i, text=i, type=VAR
value==, text==, type=ASSIGN
value=0, text=0, type=INT
value=;, text=;, type=SEMICOLON
value=i, text=i, type=VAR
value=<, text=<, type=LT
value=n, text=n, type=VAR
value=;, text=;, type=SEMICOLON
value=++, text=++, type=INC
value=i, text=i, type=VAR
value=), text=), type=R_PARENTHESIS
value=for, text=for, type=KEY_WORD
value=(, text=(, type=L_PARENTHESIS
value=int, text=int, type=KEY_WORD
value=j, text=j, type=VAR
value==, text==, type=ASSIGN
value=0, text=0, type=INT
value=;, text=;, type=SEMICOLON
value=j, text=j, type=VAR
value=<, text=<, type=LT
value=n, text=n, type=VAR
value=;, text=;, type=SEMICOLON
value=++, text=++, type=INC
value=j, text=j, type=VAR
value=), text=), type=R_PARENTHESIS
value=swap, text=swap, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=a, text=a, type=VAR
value=[, text=[, type=L_BRAKET
value=i, text=i, type=VAR
value=], text=], type=R_BRAKET
value=,, text=,, type=COMMA
value=a, text=a, type=VAR
value=[, text=[, type=L_BRAKET
value=j, text=j, type=VAR
value=], text=], type=R_BRAKET
value=), text=), type=R_PARENTHESIS
value=;, text=;, type=SEMICOLON
value=for, text=for, type=KEY_WORD
value=(, text=(, type=L_PARENTHESIS
value=int, text=int, type=KEY_WORD
value=i, text=i, type=VAR
value==, text==, type=ASSIGN
value=0, text=0, type=INT
value=;, text=;, type=SEMICOLON
value=i, text=i, type=VAR
value=<, text=<, type=LT
value=n, text=n, type=VAR
value=;, text=;, type=SEMICOLON
value=++, text=++, type=INC
value=i, text=i, type=VAR
value=), text=), type=R_PARENTHESIS
value=printf, text=printf, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=%d, text="%d", type=STRING
value=,, text=,, type=COMMA
value=a, text=a, type=VAR
value=[, text=[, type=L_BRAKET
value=i, text=i, type=VAR
value=], text=], type=R_BRAKET
value=), text=), type=R_PARENTHESIS
value=;, text=;, type=SEMICOLON
value=}, text=}, type=R_BRACE

